test/exchange_wrapper_test.ts

```diff
@@ -93,14 +94,51 @@ describe('ExchangeWrapper', () => {
                 ];
                 await zeroEx.exchange.batchFillOrKillAsync(orderFillOrKillRequests, takerAddress);
             });
+            describe('order transaction options', () => {
+                let signedOrder: SignedOrder;
+                let orderFillOrKillRequests: OrderFillOrKillRequest[];
+                const fillableAmount = new BigNumber(5);
+                beforeEach(async () => {
+                    signedOrder = await fillScenarios.createFillableSignedOrderAsync(
+                        makerTokenAddress, takerTokenAddress, makerAddress, takerAddress, fillableAmount,
+                    );
+                    orderFillOrKillRequests = [
+                        {
+                            signedOrder,
+                            fillTakerAmount: new BigNumber(0),
+                        },
+                    ];
+                });
+                it('should validate when orderTransactionOptions are not present', async () => {
+                    return expect(zeroEx.exchange.batchFillOrKillAsync(orderFillOrKillRequests, takerAddress))
+                        .to.be.rejectedWith(ExchangeContractErrs.OrderFillAmountZero);
+                });
+                it('should validate when orderTransactionOptions specify to validate', async () => {
+                    return expect(zeroEx.exchange.batchFillOrKillAsync(orderFillOrKillRequests, takerAddress,
+                        {
+                            shouldValidate: true,
+                        },
```

Comment:
Nit: No need to put the start and end curly brackets on a new line when using an anonymous object as a method param.